react/package-lock.json

```diff
@@ -87,6 +87,45 @@
         "vite": "^6.0.7"
       }
     },
+    "../../Core-Components": {
+      "name": "@tacc/core-components",
+      "version": "0.0.3",
+      "extraneous": true,
+      "license": "MIT",
+      "dependencies": {
+        "uuid": "^8 || ^9"
+      },
+      "devDependencies": {
+        "@tacc/core-styles": "^2.25.1",
+        "@testing-library/jest-dom": "^5.17.0",
+        "@testing-library/react": "^14.0.0",
+        "@testing-library/user-event": "^14.5.2",
+        "@vitejs/plugin-react-swc": "^3.5.0",
+        "jsdom": "^22.1.0",
+        "vite": "^5.0.11",
+        "vite-plugin-dts": "^2.3.0",
+        "vite-plugin-lib-inject-css": "^2.1.1",
+        "vite-tsconfig-paths": "^4.2.0",
+        "vitest": "0.34.6"
+      },
+      "optionalDependencies": {
+        "@storybook/addon-essentials": "^7.6.19",
+        "@storybook/core-server": "^7.6.19",
+        "@storybook/react-vite": "^7.6.19",
+        "storybook": "^7.6.19"
+      },
+      "peerDependencies": {
+        "formik": "^2.2.9",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-dropzone": "^14.2.3",
+        "react-resize-detector": "^7.1.2",
+        "react-router-dom": "^6.11.2",
+        "react-step-wizard": "^5.3.11",
+        "react-table": "^7.8.0",
+        "reactstrap": "^9.1.5"
```

Comment:
Btw, if we're going to continue using core-components, we should really try to limit strict versioned dev dependency lists where we can. When I was trying to upgrade packages, I couldn't upgrade many of them because they conflicted with Core-Components dependency versions

Reply:
core-components has the following now:

```  

"peerDependencies": {
    "formik": "^2.2.9",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-dropzone": "^14.2.3",
    "react-resize-detector": "^7.1.2",
    "react-router-dom": "^6.11.2",
    "react-step-wizard": "^5.3.11",
    "react-table": "^7.8.0",
    "reactstrap": "^9.1.5"
  },```

so we would give a larger range there, right?